frontend/src/views/TestsDashboard.vue

```diff
@@ -327,10 +327,10 @@ export default {
         type: "line",
         name: `${testId}`,
         data: zip(
-          this.loadOption ? this.loadData[testId][this.loadOption] : [],
           this.displacementOption
             ? this.displacementData[testId][this.displacementOption]
-            : []
+            : [],
+          this.loadOption ? this.loadData[testId][this.loadOption] : []
```

Comment:
Isn'it more readable to write: `this.loadData?.[testId][this.loadOption] ?? [];`